pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -171,18 +171,6 @@ func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 
 	// Register event handlers to update task info in PodLister & nodeMap
 	ssn.AddEventHandler(&framework.EventHandler{
-		AllocateFunc: func(event *framework.Event) {
-			pod := pl.UpdateTask(event.Task, event.Task.NodeName)
-
-			nodeName := event.Task.NodeName
-			node, found := nodeMap[nodeName]
-			if !found {
-				klog.Warningf("node order, update pod %s/%s allocate to NOT EXIST node [%s]", pod.Namespace, pod.Name, nodeName)
-			} else {
-				node.AddPod(pod)
-				klog.V(4).Infof("node order, update pod %s/%s allocate to node [%s]", pod.Namespace, pod.Name, nodeName)
-			}
-		},
 		DeallocateFunc: func(event *framework.Event) {
```

Comment:
@Monokaix Don't we need to remove the DeallocateFunc?

Reply:
yeah it should be removed too.